src/style/PokemonDetailData.css

```
@@ -0,0 +1,64 @@
+.detail-container{
+  width: 650px;
+  height: 450px;
+  margin: 50px;
+  border: 2px solid black;
+  border-radius: 20px;
+}
+.detail-name{
```

Add space after the name.
```suggestion
.detail-name {
```